app/__init__.py

```diff
@@ -11,11 +13,16 @@ def create_app(config):
     app.config.from_object(config)
     app.json_encoder = CustomJSONEncoder
 
+    from noise import noise_blueprint
+    from co2 import co2_blueprint
     
     db.init_app(app)
 
-    from noise import noise_blueprint
+    
 
     app.register_blueprint(noise_blueprint)
+ 
```

Comment:
Newline WHY?

Reply:
🔥 app/init updated, i romoved the blank line n°24

Reply:
❤️ feature CO2 ✔️